Devscord.DiscordFramework/Framework/Commands/Parsing/CommandParser.cs

```diff
@@ -110,14 +110,24 @@ private DiscordRequestArgument GetOneArg(ref string trimmedMessage, string prefi
             arg.Name = trimmedMessage[..lastArgNameIndex];
 
             trimmedMessage = trimmedMessage.CutStart(arg.Name).TrimStart();
-            if (!trimmedMessage.StartsWith(prefix))
+            if (!trimmedMessage.StartsWith(prefix) || IsFirstArgNegativeNumber(trimmedMessage))
             {
                 arg.Value = this.GetValue(trimmedMessage, out var nextIndexAfterValue);
                 trimmedMessage = trimmedMessage[nextIndexAfterValue..].TrimStart();
             }
             return arg;
         }
 
+        private bool IsFirstArgNegativeNumber(string trimmedMessage)
+        {
+            var firstValueLength = trimmedMessage.Contains(' ') 
+                ? trimmedMessage.IndexOf(' ') 
+                : trimmedMessage.Length;
+
+            var firstValue = trimmedMessage[..firstValueLength];
+            return firstValue.Length > 1 && firstValue[0] == '-' && char.IsDigit(firstValue.CutStart("-")[0]);
```

Comment:
co jeśli chcielibyśmy inny prefix niż `-`?

Reply:
To nie jest żaden prefix tylko matematyczny minus. Liczby ujemne się zapisuje za pomocą minusa. Raczej byłoby to dziwne jakby można byłoby zapisywać liczby w taki sposób np: **+28, +56, %78**